### PR TITLE
Fixed 009 - Moved all Word documents to mark down

### PR DESCRIPTION
**Pull request by gfilicetti**
In depth review please.

### CODE REVIEW

**Comment by jrzyshr** [comment]
1) Links to challenge MD files from the main TOC are broken.  (no hyphen in the links)
2) Same problem with links in Coach Guide TOC.
3) Why are the two root readmes all caps?  I know we're not consistent, but the majority seem to be lowercase.  I think that should be our standard for new readme files going forward.
4) You created the /Student/Resources folder.   But in Coach, there is no "Solutions" folder.  Consider putting the /Coach/Code/ and /Coach/Scripts folder in a new /Coach/Solutions/ folder?  (I assume this would make more links have to be updated. However, this would be a low priority fix IMO.
5) On the hack homepage, you left the "(Optional)" label on "Repository Contents" header!

Looks great overall!!!  Thanks for the updates!

**Comment by jrzyshr** [comment]
Here's another comment.  Does this count as a review?

File: 009-MicroservicesInAzure/README.md [hunk 1]
```diff
@@ -1,20 +1,43 @@
-# Microservices in Azure
-
-This challenged based Azure Hackathon is intended to teach how to host, operate, and monitor Azure workloads running Microservices. During these three days you will be working Azure App Service, Container Instances, CosmosDB, and Application Insights.
-
-[Proctors Guide](https://github.com/microsoft/WhatTheHack/blob/master/009-MicroservicesInAzure/Host/Guides/MicroservicesOnAzure-ProctorGuide.docx?raw=true)
-
-[Hack Day Presentations - Running the Hack](https://github.com/microsoft/WhatTheHack/blob/master/009-MicroservicesInAzure/Host/Guides/Lectures.pptx?raw=true)
-
-[Student Guid](https://github.com/microsoft/WhatTheHack/blob/master/009-MicroservicesInAzure/Student/Guides/MicroservicesOnAzure-StudentGuide.pdf?raw=true)
```
Comment (andywahr) [inline 009-MicroservicesInAzure/README.md:9]: this should be Student Guide (spelling)

**Comment by gfilicetti** [comment]
@jrzyshr The standard the world over for readme files is all CAPS.... just look at what you get when you create a new repo in GitHub. Comes from an ancient standard in Unix and C programming having "README" files (no extension) for calling people's attention.

We should flag all PRs that submit readme's that aren't NOT all caps

**Comment by gfilicetti** [comment]
@jrzyshr Look at the latest two commits for your fixes.

**Pull request merged**